src/librustdoc/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    rustdoc::main();
+}
```

Comment:
I think`src/librustdoc` should be moved to `src/tools/rustdoc`, merging it with the already existing binary that does exactly the same as this file.

Reply:
This code is going to be reverted when I change this to use x.py instead of rustdoc. If you think rustdoc should be moved I'm not entirely opposed, but it definitely shouldn't happen here.